# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hugin/widget.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

wxw26 defect, use top level window as parent

--- hugin-0.5/src/hugin/PanoPanel.cpp_orig	2005-12-13 19:52:06.000000000 +0100
+++ hugin-0.5/src/hugin/PanoPanel.cpp	2005-12-13 19:57:03.000000000 +0100
@@ -562,7 +562,7 @@
     // select output file
     // FIXME put in right output extension for selected
     // file format
-    wxFileDialog dlg(this,_("Create panorama image"),
+    wxFileDialog dlg(NULL,_("Create panorama image"),
                      wxConfigBase::Get()->Read(wxT("actualPath"),wxT("")),
                      wxT(""), wxT(""),
                      wxSAVE, wxDefaultPosition);
